tools/downloader/converter.py

```diff
@@ -56,19 +56,26 @@ def prefixed_subprocess(prefix, args):
             sys.stdout.write(prefix + ': ' + line)
         return p.wait() == 0
 
+
 def convert_to_onnx(model, output_dir, args, stdout_prefix):
-    pytorch_converter = Path(__file__).absolute().parent / 'pytorch_to_onnx.py'
-    prefixed_printf(stdout_prefix, '========= {}Converting {} to ONNX',
-        '(DRY RUN) ' if args.dry_run else '', model.name)
-
-    pytorch_to_onnx_args = [string.Template(arg).substitute(conv_dir=output_dir / model.subdirectory,
-                                                            dl_dir=args.download_dir / model.subdirectory)
-                            for arg in model.pytorch_to_onnx_args]
-    cmd = [str(args.python), str(pytorch_converter), *pytorch_to_onnx_args]
+    script_dir = Path(__file__).absolute().parent
+    converters = {
+        'pytorch': 'pytorch_to_onnx.py',
+    }
```

Comment:
I don't think we need a separate dictionary with converters. You can replace `KNOWN_ONNX_SUPPORT` with `ONNX_CONVERTERS`, which can serve both as a set of supported frameworks and a converter dictionary.